public/components/custom_panels/custom_panel_table.tsx

```diff
@@ -120,7 +121,10 @@ export const CustomPanelTable = ({
   };
 
   const onCreate = async (newCustomPanelName: string) => {
-    if (!isNameValid(newCustomPanelName)) {
+    const nameFlag = await doesNameExist(newCustomPanelName);
+    if (await nameFlag()) {
```

Comment:
The double-await seems weird?

Also nameFlag: might tell the story better if was "duplicateFound" or sth

Reply:
removed